regex-assembly/942151.ra

```diff
@@ -5,270 +5,4 @@
 ##!^ \b
```

Comment:
Prefix and suffix belong to the word list. They need to be in the included file. Only flags don't work.

Reply:
Damn. Another bug. This one is simple to fix. I'll do that tomorrow evening.

Reply:
Ok, great (not the bug ;-) ). Thank you!

Reply:
I've created a new release of `crs-toolchain` with a fix. I hope i works now.

Reply:
Thanks a lot @theseion! The new version works and the output does not change!